code/Modules/Items/Carriables/ItemTorchlight.cs

```diff
@@ -0,0 +1,153 @@
+﻿using Sandbox;
+using ssl.Modules.Items.Data;
+
+namespace ssl.Modules.Items.Carriables
+{
+    public partial class ItemTorchlight : Item
+    {
+        private const int Range = 512;
+        private const float Falloff = 1.0f;
+        private const float LinearAttenuation = 0.0f;
+        private const float QuadraticAttenuation = 1.0f;
+        private const int Brightness = 2;
+        private const int InnerConeAngle = 20;
+        private const int OuterConeAngle = 40;
+        private const float FogStength = 1.0f;
+        private const string SlideAttachementName = "slide";
+        private const string LightAttachementName = "light";
+        private const string FlashLighOnName = "flashlight-on";
+        private const string FlashLighOffName = "flashlight-off";
```

Comment:
Missing "t" and we should know that this is a sound name
(I cannot add suggestions i'm in the train)